src/components/network-chart/index.css

```diff
@@ -90,14 +80,11 @@
 .networkChartRowItemUriRequired {
   display: inline-block;
   overflow: hidden;
+  max-width: 600px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
 }
 
-.networkChartRowItemUriRequired:nth-child(1) {
-  padding-right: 8px;
-}
```

Comment:
I believe this was an outdated style, as we always have the scheme (which is optional) as first child now. I think this comes from a previous version of the chart where we wouldn't always show the scheme.